osu.Game/Screens/Edit/Editor.cs

```diff
@@ -322,6 +322,13 @@ private void load(OsuConfigManager config)
                                                 State = { BindTarget = editorHitMarkers },
                                             }
                                         }
+                                    },
+                                    new MenuItem("Timing")
+                                    {
+                                        Items = new MenuItem[]
+                                        {
+                                            new EditorMenuItem("Set Current Position as Preview Point", MenuItemType.Standard, SetCurrectTimeAsPreview)
```

Comment:
none of the other menu items are cased like this. i also don't know why this is talking about "position"

```suggestion
                                            new EditorMenuItem("Set current time as preview point", MenuItemType.Standard, SetCurrectTimeAsPreview)
```

Reply:
FWIW that is stable's wording/casing, but I agree with the suggestion above.

Reply:
Maybe this needs to be discussed? Because I'm doing this to correspond with stable